cmd/tempo-vulture/main.go

```diff
@@ -115,6 +115,11 @@ func queryTempoAndAnalyze(baseURL string, backoff time.Duration, traceIDs []stri
 		glog.Error("tempo url ", baseURL+"/api/traces/"+id)
 		trace, err := util.QueryTrace(baseURL, id, tempoOrgID)
 		if err != nil {
+			if strings.Contains(err.Error(), "not found") {
```

Comment:
It would be preferred to define and export an error from the util package like:

```
var ErrTraceNotFound = errors.New("trace not found")
```

The calling code can then compare using simple equality instead of a `string.Contains` method

```
if err == util.ErrTraceNotFound {
}
```

Reply:
+1

Reply:
Thanks for the review! Yep, that way it's cleaner. I'll update the PR :)